internal/controller/database_controller.go

```diff
@@ -113,8 +113,8 @@ func (r *DatabaseReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 		}
 	}()
 
+	r.Recorder.Event(dbcr, "Normal", phase, "Started reconciling db-operator managed databases")
```

Comment:
I'd change `databases` to `database`, cause one reconciliation is taking care of one dbcr

Reply:
fixed